src/js/content.js

```diff
@@ -60,21 +61,32 @@ function openPopup(payload) {
   if (!document.getElementById("moco-bx-popup-root")) {
     const domRoot = document.createElement("div")
     domRoot.setAttribute("id", "moco-bx-popup-root")
-    document.body.appendChild(domRoot)
+    const rootNode = document.querySelector("[aria-modal]") || document.body
```

Comment:
Can we check if the service is `trello` and otherwise always attach to the `document.body`?

Reply:
If so, I think we need to address those separately. I am afraid this could break other services.

Reply:
I tested Jira and it also works attaching the popup to aria-modal. Let's go with this.